pages/docs/installation/installation-special-systems.md

```diff
@@ -16,6 +16,52 @@ The instructions may still be valuable for unlisted systems.
 
 ## Active systems
 
+### HAWK (HPE Apollo/AMD, Stuttgart)
+
+#### Building
+
+The following steps explain how to install preCICE on HAWK with PETSc and MPI (using the system standard MPE):
+(1) [Download Eigen](http://eigen.tuxfamily.org/index.php?title=Main_Page) and copy it to HAWK. Afterwards export the `EIGEN3_ROOT`, e.g.,
+
+```bash
+export EIGEN3_ROOT="$HOME/precice/eigen"
+```
+
+(2) Load available modules:
+
+```bash
+module load cmake boost petsc/<VERSION>-int32-shared
+```
+
+(3) Build preCICE. The library and include path for PETSc need to be defined explicitly (for an unknown reason):
+
+```bash
+cmake -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX="my/install/prefix" -DPRECICE_PETScMapping=ON -DPETSc_INCLUDE_DIRS="$PETSC_DIR/include" -DPETSc_LIBRARIES="$PETSC_DIR/lib/libpetsc.so" -DPRECICE_PythonActions=OFF .. /path/to/precice/source
+
+make install -j 16
+```
+
+{% include important.html content="Running the tests for the installation will fail, even though the installation is fine, since the tests require MPI facilities which need to be enabled explicitly." %}
```

Comment:
I am not sure what you mean here with "MPI facilities" (is this the right term?). It would be useful to give a short hint/link on how to enable this.

Reply:
I don't know it actually. The tests require to enable `MPI spawning` and ofter some trials I gave it up since I don't see any benefit from making the tests run.

Reply:
Then maybe write `since the tests require MPI features that are disabled by default`.

Reply:
Done